File: app/src/main/java/com/devdream/cookall/core/services/recipe/RecipeRetrofitService.java
package com.devdream.cookall.core.services.recipe;

import com.devdream.cookall.core.api.responses.RecipeListResponse;

import retrofit2.Call;
import retrofit2.http.GET;

interface RecipeRetrofitService {

    @GET("/recipes")
    Call<RecipeListResponse> fetchAllRecipes();

}

File: app/src/main/java/com/devdream/cookall/core/services/recipe/RecipeRetrofitService.kt
package com.devdream.cookall.core.services.recipe

import com.devdream.cookall.core.api.responses.RecipeListResponse

import retrofit2.Call
import retrofit2.http.GET

internal interface RecipeRetrofitService {

    @GET("/recipes")
    fun fetchAllRecipes(): Call<RecipeListResponse>

}
